### DISCUSSION TITLE
太陽能電池×AI

### DISCUSSION THREAD

**Discussion by stan890425** [1101-GS4719-C/Python-Community-GS4719-C#554] [科技新知]
簡單介紹太陽能電時的發展, 太陽能電池大致可分為三代
第一代太陽能電池 也就是矽晶太陽能電池 ，但成本昂貴:money_with_wings:
第二代太陽能電池 為無機材料為主的太陽能電池  成本相對較低 但缺點就是重金屬毒性較高:skull: 
第三代太陽能電池 與前兩代最大的不同是以有機材料為主的太陽能電池 
有光伏打電池 染料敏化電池 以及鈣鈦礦太陽能電池 相對於前兩代，
這種有機太陽能電池成本低廉,但光電轉換效率不如前兩代的好,是我們必須克服得難關
其中鈣鈦礦電池為第三代太陽能電池中最高效率的，但因其製備過程中需要不斷的重複試條件，
且在不同的大小下會有不同的最佳條件，因此這篇報導指出 現在可以用AI模擬，以減少人力消耗及計算。

文章來源:https://pansci.asia/archives/345195

**Comment by ThomasMeng0515** [comment]
鈣鈦礦的組成成分很複雜，要經過很多實驗才能找到最佳比例，利用機器學習模型去模擬與預測降低資源消耗並且提升研發速度。期望日後真的可以利用AI成功提升太陽能電池的光電轉換效率。

**Comment by sean1325418** [comment]
現在開發新材料都能先利用AI幫助計算，這樣能減少很多的開發成本，希望也能讓價格變低一點

**Comment by PoooYen** [comment]
氣候條件的多變非常適合用強大的AI來進行像這樣的多變數運算!
希望之後能有能源轉換效率50%以上的商用太陽能電池出現

**Comment by Sazmafuji** [comment]
能夠更加提升太陽能板吸收率增加，希望能盡快解決缺電

**Comment by awkward-willy** [comment]
未來感覺許多研發工作都能透過 AI 去模擬跟計算
來減少很多人工試驗的時間
同時，若能設計出更環保且高效的電池
一定能讓整體地球環境更進一步

**Comment by Rosancl** [comment]
相信未來能透過AI模擬應用的層面跟領域會越來越多，能夠減少很多開發成本。

**Comment by Lai27330** [comment]
透過AI模擬計算，可以有效縮減製造太陽能電池大量時間及成本，對於幾乎一年四季都不缺陽光的地方很有幫助。

**Comment by treeann** [comment]
不管什麼領域AI運算都很重要呢

**Comment by Yuhsin000** [comment]
多變且複雜的條件，很難預測及評估效用，AI應該會很有幫助

**Comment by Jimmy911218** [comment]
氣候的複雜性非常適合ai來演算，能大大減少成本

**Comment by leon10456** [comment]
感謝樓主分享 不管什麼領域AI運算都很重要呢

**Comment by liangyinglly** [comment]
看來材料系或地科系都要與AI共存了

**Comment by HaoFish123** [comment]
感謝分享，看來AI的應用範圍又更廣了。

**Comment by HsuChen88** [comment]
用AI先模擬情境已是常用的實驗或訓練方法

**Comment by yucheng1028** [comment]
ai應用真的好廣

**Comment by alexthebigpp** [comment]
感謝分享，又上了一課。

**Comment by yenwei12138** [comment]
這種需要大量精密計算的工作，真的挺適合AI，又持久又比人力精準
